taca/nanopore/ont_transfer.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import shutil
+
+from taca.nanopore.nanopore import Nanopore
+from taca.utils.config import CONFIG
+
+logger = logging.getLogger(__name__)
+
+
+class ONTTransfer(Nanopore):
+    """ONT run for transfer to HPC cluster"""
```

Comment:
isn't PromethION also an ONT run? Should this be for MinION?

Reply:
Since MinION and PromethION runs are both in the same format on ngi-nas, just in different locations, I've made the ONTTransfer class that has a different version of `archive_run` than the base class Nanopore, and two specific classes, PromethionTransfer and MinionTransfer (further down) that contain the specific location of data from each sequencer on ngi-nas. Does that make sense?

Reply:
Ah, yes. Maybe the description of ONTTransfer can be rephrased to better reflect that?

Reply:
`Base class for ONT run ...` perhaps?

Reply:
Fixed 👍